centreon/src/Core/Security/ProviderConfiguration/Infrastructure/Repository/HttpReadAttributePathRepository.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Security\ProviderConfiguration\Infrastructure\Repository;
+
+use Core\Security\Authentication\Domain\Exception\SSOAuthenticationException;
+use Core\Security\ProviderConfiguration\Domain\Exception\Http\InvalidContentException;
+use Core\Security\ProviderConfiguration\Domain\Exception\Http\InvalidResponseException;
+use Core\Security\ProviderConfiguration\Domain\Exception\Http\InvalidStatusCodeException;
+use Core\Security\ProviderConfiguration\Domain\LoginLoggerInterface;
+use Core\Security\ProviderConfiguration\Domain\Model\Configuration;
+use Core\Security\ProviderConfiguration\Domain\Repository\ReadAttributePathRepositoryInterface;
+use Exception;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\RedirectionExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+class HttpReadAttributePathRepository implements ReadAttributePathRepositoryInterface
+{
+    /**
+     * @param HttpClientInterface $client
+     */
+    public function __construct(private readonly HttpClientInterface $client)
+    {
+    }
+
+    /**
+     * @param string $url
+     * @param string $token
+     * @param Configuration $configuration
+     * @return array
+     * @throws ClientExceptionInterface
```

Comment:
Check php-cs-fixer for all modified files, I think it adds blank lines between `@param`, `@throws`, ...